smailyforprestashop.php

```diff
@@ -280,6 +309,9 @@ public function getContent()
         $cart_autoresponder_for_template = pSQL((Configuration::get('SMAILY_CART_AUTORESPONDER')));
         $cart_autoresponder_for_template = str_replace('\"', '"', $cart_autoresponder_for_template);
         $cart_autoresponder_for_template = unserialize($cart_autoresponder_for_template);
+        //
+        $categories = Category::getNestedCategories(null, Context::getContext()->language->id);
+        //
```

Comment:
Rather than using comments to improve readability, just use whitespace.

```suggestion

        $categories = Category::getNestedCategories(null, Context::getContext()->language->id);

```

Reply:
Added